src/adapter/src/catalog/apply.rs

```diff
@@ -1289,10 +1313,18 @@ impl CatalogState {
                     let span = info_span!(parent: None, "parse builtin view", name = view.name);
                     OpenTelemetryContext::obtain().attach_as_parent_to(&span);
                     let task_state = Arc::clone(&spawn_state);
+                    let cached_expr = local_expression_cache.remove_cached_expression(&id);
```

Comment:
Why are the expressions removed from the cache? Because we expect each one to be used at most once?

Reply:
Yes and I didn't want to clone every expression.